platforms/documentation/docs/src/snippets/bestPractices/useBuildCache-avoid/groovy/build.gradle

```
@@ -0,0 +1,5 @@
+// tag::avoid-this[]
+plugins {
+    id 'java'
```

```suggestion
    id 'java-library`
```

Library should be enough.  `java` should really be called `java-application`.

---

Library should be enough.  `java` should really be called `java-application`.